packages/itwinui-css/src/tag/tag.scss

```diff
@@ -37,6 +36,11 @@
   &:hover {
     border-color: var(--iui-color-border-foreground-hover);
   }
+
+  &:focus {
```

Comment:
Using `:focus` instead of `@mixin iui-focus` because I want to show the selection when clicked on.